packages/web/components/Player/Filter/PlayerFilter.tsx

```diff
@@ -122,86 +125,273 @@ export const PlayerFilter: React.FC<Props> = ({
+          alignItems="center"
+          justifyContent="center"
+          flexWrap={{
+            base: 'nowrap',
+            md: 'wrap',
+            '2xl': 'nowrap',
+          }}
+          flexDirection={{
+            base: 'row',
+            md: 'column',
+            '2xl': 'row',
+          }}
         >
-          <InputGroup size="lg">
-            <Input
-              background="dark"
-              w="100%"
-              type="text"
-              minW={{
-                base: 'min(18rem, calc(100vw - 2rem))',
-                sm: 'md',
-                md: 'lg',
-                lg: 'xl',
-              }}
-              placeholder="SEARCH PLAYERS BY USERNAME OR ETHEREUM ADDRESS"
-              _placeholder={{ color: 'whiteAlpha.500' }}
-              value={search}
-              onChange={(e) => setSearch(e.target.value)}
-              size="lg"
-              borderRadius={10}
-              borderColor="borderPurple"
-              fontSize="md"
-              borderWidth="2px"
-            />
-            {search.length > 0 && (
-              <InputRightElement>
-                <IconButton
-                  p="2"
-                  variant="link"
-                  colorScheme="white"
-                  icon={<CloseIcon />}
-                  onClick={() => {
-                    setSearch('');
-                    setQueryVariable('search', '%%');
+          <Box
+            flexGrow={{
+              base: '2', // expand to take up more space when side-by-side with filters button
+              md: '0',
+            }}
+            marginRight={{
+              base: '4',
+              md: '0',
+              '2xl': '6',
+            }}
+            marginBottom={{
+              base: '0',
+              md: '4',
+              '2xl': '0',
+            }}
+            maxW={{
+              base: 'sm',
+              md: '100%',
+              '2xl': 'sm',
+            }}
+          >
+            {/**
+             * SEARCH BOX
+             * Initial minWidth of the search input is 14em, so it will fit on a tiny screen.
+             * After md breakpoint, the minwidth is 30em. (Closer match to the dropdowns while in a stacked layout)
+             * After 2xl breakpoint, the minWidth is 20em so it will fit in a single row with the filter dropdowns.
+             * There's no search button any more; it's set up to search onChange once the search string is longer than 2 chars.
+             */}
+            <Form width="fill" onSubmit={onSearch} display="flex">
+              <InputGroup>
+                <InputLeftElement
+                  pointerEvents="none"
+                  children={<SearchIcon />}
+                />
+                <Input
+                  background="dark"
+                  borderColor="borderPurple"
+                  borderLeftRadius={4}
+                  borderRightRadius={{
+                    base: '4',
+                    md: '0',
                   }}
```

Comment:
```suggestion
                  borderRightRadius={{ base: '4', md: '0' }}
```

Reply:
The borderRadius should be 4 all around. (We took out the search button that used to be glued to the right side of the search input above the md breakpoint)